apps/dashboard/app/(app)/semantic-cache/settings/page.tsx

```diff
@@ -0,0 +1,68 @@
+import { CopyButton } from "@/components/dashboard/copy-button";
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "@/components/ui/card";
+import { Code } from "@/components/ui/code";
+import { getTenantId } from "@/lib/auth";
+import { db } from "@/lib/db";
+import { redirect } from "next/navigation";
+import { DeleteGateway } from "./delete-gateway";
+
+export default async function SemanticCacheSettingPage() {
+  const tenantId = getTenantId();
+  const workspace = await db.query.workspaces.findFirst({
```

Comment:
you probably need to load the llm gateway by id
right now you're assuming every user has one at most

Reply:
We should be able to assume that, at the moment it's not possible to create multiple gateways. If the user has already created a gateway, /semantic-cache/new will redirect them